packages/warehouses/src/warehouseClientFromCredentials.ts

```diff
@@ -1,16 +1,17 @@
 import {
     CreateWarehouseCredentials,
-    UnexpectedServerError,
     WarehouseTypes,
+    UnexpectedServerError,
 } from 'common';
-import Logger from '../../logger';
-import { WarehouseClient } from '../../types';
-import BigqueryWarehouseClient from './BigqueryWarehouseClient';
-import DatabricksWarehouseClient from './DatabricksWarehouseClient';
-import PostgresWarehouseClient from './PostgresWarehouseClient';
-import RedshiftWarehouseClient from './RedshiftWarehouseClient';
-import SnowflakeWarehouseClient from './SnowflakeWarehouseClient';
+import * as Logger from 'winston';
```

Comment:
Is this working as expected ? is it picking up the configuration from the backend ?

Reply:
just testing